android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java

```diff
@@ -49,8 +49,7 @@
     @BindView(R.id.list_sponsors)
     RecyclerView sponsorsRecyclerView;
 
-    private RealmDataRepository realmRepo = RealmDataRepository.getDefaultInstance();
-    private RealmResults<Sponsor> realmResults;
+    SponsorsFragmentViewModel sponsorsFragmentViewModel;
```

Comment:
This instance should be private instead of package-private. Isn't it ?

Reply:
Initially it was private but then I got an error from codacy so I made this change

Reply:
Make this local